WalletCore/Classes/Core/Repository/LWRxAuthManagerCashOutSwift.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  LWRxAuthManagerCashOutSwift.swift
+//  WalletCore
+//
+//  Created by Nacho Nachev on 2.11.17.
+//  Copyright © 2017 Lykke. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+
+public class LWRxAuthManagerCashOutSwift: LWRxAuthManagerBase<LWPacketCashOutSwift> {
+
+    public func request(withData data: LWPacketCashOutSwift.Body) -> Observable<ApiResult<Void>> {
+        return Observable.create { observer in
+            let packet = LWPacketCashOutSwift(body: data, observer: observer)
+            GDXNet.instance().send(packet, userInfo: nil, method: .REST)
+            
+            return Disposables.create {}
+        }
+            .startWith(.loading)
+            .shareReplay(1)
+    }
+    
+    override func onError(withData data: [AnyHashable : Any], pack: LWPacketCashOutSwift) {
+        guard let observer = pack.observer as? AnyObserver<ApiResult<Void>> else {return}
+        
+        observer.onNext(.error(withData: data))
+        observer.onCompleted()
+    }
+    
+    override func onSuccess(packet: LWPacketCashOutSwift) {
+        guard let observer = packet.observer as? AnyObserver<ApiResult<Void>> else {return}
+        
+        observer.onNext(.success(withData: Void()))
+        observer.onCompleted()
+    }
+    
+    override func onForbidden(withPacket packet: LWPacketCashOutSwift) {
+        guard let observer = packet.observer as? AnyObserver<ApiResult<LWPacketCashOutSwift>> else {return}
```

Comment:
Since produced observer in request method is of type AnyObserver<ApiResult<Void>>, this statement will always be false and following code will be never executed.